lib/src/button/button.component.html

```diff
@@ -0,0 +1,46 @@
+<ng-container [ngSwitch]="tag">
+    <input
+        [type]="type"
+        *ngSwitchCase="'input'"
+        [ngClass]="theme | buttonClasses: size:block:className"
+        (click)="handleClick($event)"
+        [disabled]="disabled"
+        [attr.id]="id"
+        [attr.name]="name"
+        [value]="label"
+        [attr.aria-disabled]="disabled"
+        [attr.aria-pressed]="active"
+        [class.active]="active"
+    />
+
+    <a
+        href="#"
```

Comment:
I would like an option to pass the href if I am using this component as an anchor tag